src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 30px auto;
```

Comment:
BEM does not allow us to use margin and task requires `padding 10px`

Reply:
Solved.